search-app/src/cmr/search/services/acls/draft_acls.clj

```diff
@@ -0,0 +1,26 @@
+(ns cmr.search.services.acls.draft-acls
+  "Contains functions for manipulating draft acls"
+  (:require
+   [clojure.string :as string]
+   [cmr.common-app.services.search.group-query-conditions :as gc]
+   [cmr.common-app.services.search.query-execution :as qe]
+   [cmr.common-app.services.search.query-model :as qm]
+   [cmr.common.concepts :as cc]
+   [cmr.search.services.acls.acl-helper :as acl-helper]
+   [cmr.transmit.config :as tc]))
+
+(doseq [concept-type (cc/get-generic-concept-types-array)]
+  (when (string/includes? (name concept-type) "draft")
```

Comment:
heres an is-draft-concept? function in common-lib/src/cmr/common/concepts.clj we could use here

Reply:
Done.